helmfile/charts/grafana-ops/templates/dashboard-configmap .yaml

```
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-dashboards
+  namespace: monitoring
+  labels:
+    {{ .Values.labelKey }}: 'dashboards'
+data:
+{{- if .Values.dashboards.backup.enabled }}
+  backup-dashboard.json: |-
+    {{- .Files.Get "dashboards/backup-dashboard.json" | nindent 4 }}
+{{- end }}
+{{- if .Values.dashboards.elasticsearch.enabled }}
+  elasticsearch-dashboard.json: |-
+    {{- .Files.Get "dashboards/es-dashboard.json" | nindent 4 }}
+{{- end }}
+{{- if .Values.dashboards.falco.enabled }}
+  falco-dashboard.json: |-
+    {{- regexReplaceAll "kibanaURL" (.Files.Get "dashboards/falco-dashboard.json") .Values.dashboards.falco.logEndpoint | nindent 4 }}
```

I guess this is fine, but it would feel better if we could wrap it inside like  `<<>>` or something.

```suggestion
    {{- regexReplaceAll "<<kibanaURL>>" (.Files.Get "dashboards/falco-dashboard.json") .Values.dashboards.falco.logEndpoint | nindent 4 }}
```

Just to be more safe that we don't accidentally replaces something

---

Good point, changed now.